### ISSUE TITLE
Debugger entered--Lisp error: (void-function benchmark-init/show-durations-tabulated)

### ISSUE THREAD

**Issue by mcandre** [dholm/benchmark-init-el#5]
I'm trying to follow the instructions for setting up benchmark-init, but I can't tabulate the benchmark results.

**Download**

```
$ cd ~/Desktop/src/
$ git clone git@github.com:dholm/benchmark-init-el.git
```

**Configuration**

https://github.com/mcandre/dotfiles/blob/benchmark/.emacs

I relaunched Emacs. Twice.

Then I tried to run `(benchmark-init/show-durations-tabulated)`, Emacs complains that it can't find this function. Tab completion shows several functions for `M-: (benchmark-init/...`, but `show-durations-tabulated` is not one of them.

**System**

```
$ specs emacs brew os
Specs:

specs 0.12
https://github.com/mcandre/specs#readme

emacs --version
GNU Emacs 24.4.1
Copyright (C) 2014 Free Software Foundation, Inc.
GNU Emacs comes with ABSOLUTELY NO WARRANTY.
You may redistribute copies of Emacs
under the terms of the GNU General Public License.
For more information about these matters, see the file named COPYING.

brew --version
0.9.5

system_profiler SPSoftwareDataType | grep 'System Version'
      System Version: OS X 10.10.2 (14C109)
```

Hmm, does benchmark-init play well with use-package, especially delayed package loading?

**Comment by dholm** [comment]
@mcandre Sorry for the late reply, it seems that I've been having some problems with GitHub email notifications.

It should work with `use-package` and delayed loading as long as you activate `benchmark-init` early on.  I was able to reproduce your problem using your configuration file and I realized that I have misunderstood how the autoload cookies work.  I have pushed a fix for this, update your benchmark-init and then run `make` in it.  Then change the top two lines in your configuration to:

    (add-to-list 'load-path "~/Desktop/src/benchmark-init-el/")
    (require 'benchmark-init-loaddefs)
    (benchmark-init/activate)
